rootfs/etc/services.d/uat2esnt/run

```diff
@@ -8,5 +8,5 @@ sleep 5
 # shellcheck disable=SC2016
 socat -u "TCP:127.0.0.1:30978" STDOUT | \
 uat2esnt | \
-socat -d -d -u STDIN "TCP-LISTEN:37981,fork" \
+nc -lkp 37981 \
```

Comment:
this was suggested on discord by @kx1t: with socat, output is consumed by the first client that reads it, whereas with ncat all output is duplicated to all clients.

This should probably be changed in all containers using socat like this.

Reply:
I think it was @fredclausen  who suggested that, but all good :)

Reply:
did a search - @wiedehopf wins!